tests/phpunit/elementor/modules/atomic-widgets/test-atomic-widget-base.php

```diff
@@ -720,6 +720,30 @@ public function test_get_data_for_save() {
 							'background' => [
 								'$$type' => 'background',
 								'value' => [
+									'background-overlay' => [
```

Comment:
For Elena's PR we seem to be using different tests. Can you please check the comments in this task: https://elementor.atlassian.net/browse/ED-17833

Reply:
@Batsirai-muchareva Yael and Nami are adding tests to Lena's PRs so that it can be merged.

Reply:
I was slightly mistaken about the tests.

By point still stands in general. Can you please check the instructions on the ticket mentioned above about which tests need to be added?